hack/arktos-up-scale-out-poc.sh

```diff
@@ -640,12 +640,8 @@ if [[ "${START_MODE}" != "nokubelet" ]]; then
         KUBELET_LOG=""
         ;;
       Linux)
-    	if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
-	   KUBELET_LOG=/tmp/kubelet.log
-           kube::common::start_kubelet
-	else
-           KUBELET_LOG=""
-    	fi
+	       KUBELET_LOG=/tmp/kubelet.log
```

Comment:
If the indent is adjusted, that will be better.

Reply:
done